bin/smoke-test-registry.sh

```diff
@@ -6,6 +6,7 @@ for ((i = 0; i <= retries; i++)); do
     response=$(
         curl --silent --verbose --fail --request POST \
             --header 'Content-Type: application/vnd.schemaregistry.v1+json' \
+            --header 'Authorization: Basic Og==' \
```

Comment:
This are empty credentials, i.e. `username="" | password=""`, this is needed as now on FastAPI we require the credentials to be at least present, even if it's empty, we can revise this later as it'd be now a breaking change, but shouldn't hurt too much.